SitumWayfinding/Classes/Positioning/PositioningViewController.swift

```diff
@@ -208,11 +216,10 @@ class PositioningViewController: UIViewController ,GMSMapViewDelegate, UITableVi
     }
     
     func initializeNavigationBar() {
-        navbar.topItem?.title = buildingInfo!.building.name
+        //navbar.topItem?.title = buildingInfo!.building.name
```

Comment:
No longer needed.. we should remove it then.. Who is updating this component now?